consumer.py

```diff
@@ -1,29 +1,38 @@
-from robocorp.tasks import task
+import requests
 from robocorp import workitems
+from robocorp.tasks import task
 
-import requests
 
 @task
 def consume_traffic_data():
     """
-        Inhuman Insurance, Inc. Artificial Intelligence System robot.
-        Consumes traffic data work items.
+    Inhuman Insurance, Inc. Artificial Intelligence System robot.
+    Consumes traffic data work items.
     """
     process_traffic_data()
 
+
 def process_traffic_data():
     for item in workitems.inputs:
         payload = item.payload
         traffic_data = payload["traffic_data"]
         valid = validate_traffic_data(traffic_data)
-        if(valid):
+        if valid:
             status, return_json = post_traffc_data_to_sales_system(traffic_data)
             if status == 200:
                 item.done
```

Comment:
I assume this was meant to be a function call? Currently does nothing

Reply:
yes